api/MIDIInput.json

```diff
@@ -51,6 +51,7 @@
       "midimessage_event": {
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/MIDIInput/midimessage_event",
+          "spec_url": "https://webaudio.github.io/web-midi-api/#dom-midimessageevent",
```

Comment:
This URL is for the `MIDIMessageEvent` interface, the same as https://webaudio.github.io/web-midi-api/#midimessageevent-interface which is already linked in MIDIMessageEvent.json.

Unfortunately, there is nothing linkable for the concept of the "midimessage" event firing, it's just an algorithm in https://webaudio.github.io/web-midi-api/#dom-midiinput-onmidimessage.

I realize this is a pedantic distinction, and not one that's actually useful to make in documentation, but again see https://github.com/mdn/browser-compat-data/issues/7545 about the current mess. I'm thinking at least it's good to be consistent about what the "midimessage_event" vs. "onmidimessage" entry means so that if such entries are later merged, differences between them are clues that there's something worth noting for web developers. (Search for "addEventListener" in Document.json for attempts to capture issues like this in the current structure.)

Reply:
I've removed that link for now, pending a fix to the whole situation.